src/Compilers/Core/Portable/MetadataReader/MetadataTypeName.cs

```diff
@@ -195,6 +229,34 @@ public string TypeName
             }
         }
 
+        public ReadOnlyMemory<char> NamespaceNameMemory
+        {
+            get
+            {
+                if (_namespaceNameMemory.Equals(default))
```

Comment:
💡 Consider using `ReadOnlyMemory<char>.Empty` instead of default (they are equal and the named value is clearer)